### PR TITLE
[AIG]change path for pricing

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary

- Change path for pricing to [pricing](/ai-gateway/platform/pricing/)
- Edited pricing link of logging and changelogs

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [x] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/changelogs/ai-gateway.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ entries:
   - publish_date: "2024-09-26"
     title: Pricing
     description: |-
-      * **Pricing**: Added [pricing](/ai-gateway/pricing/) for storing logs persistently.
+      * **Pricing**: Added [pricing](/ai-gateway/platform/pricing/) for storing logs persistently.
```
Comment (pedrosousa) [inline src/content/changelogs/ai-gateway.yaml]:
```suggestion
      * **Pricing**: Added [pricing](/ai-gateway/reference/pricing/) for storing logs persistently.
```
Not sure what you should change — either the links or the file location. :)

File: src/content/docs/ai-gateway/observability/logging/index.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ Your AI Gateway dashboard shows logs of individual requests, including the user
 
 You can store up to 10 million logs per gateway. If your limit is reached, new logs will stop being saved. To continue saving logs, you must delete older logs to free up space for new logs.
 
-To learn more about your plan limits, refer to [Pricing](/ai-gateway/pricing/).
+To learn more about your plan limits, refer to [Pricing](/ai-gateway/platform/pricing/).
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/observability/logging/index.mdx]:
```suggestion
To learn more about your plan limits, refer to [Pricing](/ai-gateway/reference/pricing/).
```

**Review by pedrosousa — APPROVED** [review]

File: src/content/changelogs/ai-gateway.yaml (hunk 1 above)
Comment (daisyfaithauma) [reply src/content/changelogs/ai-gateway.yaml]: Oh. How did I miss this. Thanks

Comment (daisyfaithauma) [reply src/content/changelogs/ai-gateway.yaml]: I will change the link.

File: src/content/docs/ai-gateway/reference/pricing.mdx
Comment (pedrosousa) [inline src/content/docs/ai-gateway/reference/pricing.mdx:1]: Still regarding this file, we need to:
- Create a new redirect for it.
- Update all links still pointing to the old location (namely the two links in https://github.com/cloudflare/cloudflare-docs/blob/production/src/content/docs/ai-gateway/reference/limits.mdx).

**Pull request merged**